src/EditorFeatures/CSharpTest/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -624,5 +629,36 @@ int M(int i)
     }
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        [WorkItemAttribute(36086, "https://github.com/dotnet/roslyn/issues/36086")]
+        public async Task TestSeverity()
+        {
+            var source =
+@"class Program
+{
+    int M(int i)
+    {
+        [||]switch (i)
+        {
+            case 1:
+                return 4;
+            case 2:
+                return 5;
+            case 3:
+                return 6;
+            default:
+                return 7;
+        }
+    }
+}";
+            var warningOption = new CodeStyleOption<bool>(true, NotificationOption.Warning);
+            var options = Option(CSharpCodeStyleOptions.PreferSwitchExpression, warningOption);
+            var testParameters = new TestParameters(options: options, parseOptions: TestOptions.Regular8);
+            using var workspace = CreateWorkspaceFromOptions(source, testParameters);
+            var diags = await GetDiagnosticsAsync(workspace, testParameters);
+            Assert.Single(diags);
+            Assert.Equal(DiagnosticSeverity.Warning, diags.First().Severity);
```

Comment:
could just remove the line above and have this be diags.Single().Severity.  But i don't care that much :)

Reply:
also, can we check if the diag ID is actually the one we expect here?